Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -21,48 +21,45 @@ which ArangoDB collections a given ArangoSearch View should query for documents
 and how these documents should be queried.
 
 An ArangoSearch Link is a uni-directional connection from an ArangoDB collection
-to an ArangoSearch view describing how data coming from the said collection should
-be made available in the given view. Each ArangoSearch Link in an ArangoSearch
-view is uniquely identified by the name of the ArangoDB collection it links to.
-An ArangoSearch view may have zero or more links, each to a distinct ArangoDB
-collection. Similarly an ArangoDB collection may be referenced via links by zero
-or more distinct ArangoSearch views. In plain terms any given ArangoSearch view
-may be linked to any given ArangoDB collection of the same database with zero or
-at most one link. However, any ArangoSearch view may be linked to multiple
-distinct ArangoDB collections and similarly any ArangoDB collection may be
-referenced by multiple ArangoSearch views.
-
-To configure an ArangoSearch view for consideration of documents from a given
+to an ArangoSearch View describing how data coming from the said collection
+should be made available in the given view.  Each ArangoSearch Link in an
+ArangoSearch view is uniquely identified by the name of the ArangoDB collection
+it links to.  An ArangoSearch View may have zero or more links, each to a
+distinct ArangoDB collection.  Similarly an ArangoDB collection may be referenced
+via links by zero or more distinct ArangoSearch Views.  In other words, any given
+ArangoSearch View may be linked to any given ArangoDB collection of the same
+database with zero or one link.  However, any ArangoSearch View may be linked to
+multiple distinct ArangoDB collections and similarly any ArangoDB collection may
+be referenced by multiple ArangoSearch Views.
+
+To configure an ArangoSearch View for consideration of documents from a given
 ArangoDB collection a link definition must be added to the properties of the
-said ArangoSearch view defining the link parameters as per the section
+said ArangoSearch View defining the link parameters as per the section
 [View definition/modification](#view-definitionmodification).
 
 ## Index
 
-Inverted Index is the heart of ArangoSearch. The index consists of several 
-independent segments and the index segment itself is meant to be treated as 
-a standalone index.
-
-
+Inverted Index is the heart of ArangoSearch.  The index consists of several
+independent segments and the index segment itself is meant to be treated as a
+standalone index.
 
 ## Analyzers
 
-To simplify query syntax ArangoSearch provides a concept of 
-[named analyzers](Analyzers.md) which are merely aliases for
-type+configuration of IResearch analyzers. Management of named analyzers
-is exposed via REST, GUI and JavaScript APIs.
+To simplify query syntax ArangoSearch provides a concept of [named
+analyzers](Analyzers.md) which are merely aliases for type+configuration of
+IResearch analyzers.  Management of named analyzers is exposed via REST, GUI and
```

Comment:
There is no API for configuring analyzers yet.

Reply:
If we have something written already which will be available not before 3.4.1, you may put it inside a comment. HTML comments hide it, but won't remove it from the HTML source `<!-- foobar -->`, wheras Gitbook comments get fully removed for the output `{### foobar ###}`.

Reply:
Commented out.